<!-- Make sure launch nav_preapare first -->
<launch>
    <param name="/use_sim_time" value="true" />
    
    <node type="task13.py" name="task13" pkg="thewalkingdead" output="screen">
    </node>

</launch>